packages/events/src/service/deduplication/deduplication.test.ts

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
Tests copied from legacy birth deduplication. You can see how deduplication logic is configured starting from line 24